src/R/send-charts.R

```diff
@@ -1,7 +1,11 @@
+args = commandArgs(trailingOnly=TRUE)
+output_dir = args[1]
+
 install_missing_packages <- function(package_name) {
   if(!require(package_name, character.only = TRUE)) {
     install.packages(package_name, repos='http://cran.us.r-project.org') } }
 
+# svglite requires cairo dev packages
```

Comment:
R "should" download other deps that a library has... I know it's in the README, but does it make sense to add it to this list to autodownload?

Reply:
These are OS packages...

Reply:
👍